crates/subspace-runtime/src/lib.rs

```diff
@@ -944,6 +944,8 @@ construct_runtime!(
 
         // Reserve some room for other pallets as we'll remove sudo pallet eventually.
         Sudo: pallet_sudo = 100,
+
+        // 254 is reserved for invalid_bundles_undecodeable_tx testing.
```

Comment:
The test targets the extrinsic of [the test evm runtime](https://github.com/autonomys/subspace/blob/main/domains/test/runtime/evm/src/lib.rs) not the consensus runtime.